.gitignore

```diff
@@ -13,7 +13,8 @@ out/
 .idea/sonarlint/
 .idea/modules/
 .idea/*.xml
+compose.yaml
```

Comment:
This is until we finalize the auth server image, right? Then it's worth a comment.

Reply:
hmm now this is stable - we do provide the auth-server image in both archs now.